src/msw/webview_ie.cpp

```diff
@@ -853,25 +857,190 @@ wxString wxWebViewIE::GetPageText() const
     }
 }
 
-void wxWebViewIE::RunScript(const wxString& javascript)
+bool wxWebViewIE::MSWSetEmulationLevel(bool defaultLevel)
+{
+    wxRegKey key(wxRegKey::HKCU, _T(REGISTRY_IE_PATH));
+    if ( key.Exists() )
+    {
+        if ( !defaultLevel )
+	    {
+            if ( !key.SetValue(wxGetFullModuleName().AfterLast('\\'), IE_EMULATION_LEVEL) )
+            {
+	            wxLogWarning(_("Failed to set the current browser control emulation level"));
+                return false;
+            }
+        }
+        else
```

Comment:
Please put `{` and `}` around the `else` branch if `if` has it (and vice versa too, of course).